mozilla/blurts-server
# add new plan_tier key on all glean events

# References: 
Jira: MNTOR-3116




# Description

Add `plan_tier` with value of `Free` or `Plus` to all  Glean events.

# How to test

You can see your pings on https://debug-ping-preview.firebaseapp.com/pings/fx-monitor-local-dev or in the browser devtools console.

Use the http://localhost/admin/dev UI to add/remove Plus, and ensure that pings have the correct `plan_tier`.

# Checklist (Definition of Done)
